docs/fa/docs/css/rtl.css

```diff
@@ -0,0 +1,15 @@
+.termy{
+    direction: ltr;
+}
+code{
+    direction: ltr;
+    display: inline-block;
+}
```

Comment:
Hey Sebastián,

I added this because Persian language is RTL and I made the whole document RTL. But some parts of the doc including links and codes should always be LTR, no matter what the translation language is, otherwise they will be rendered in a bad way (Not readable actually). 

Also the `display: inline-block` is required because otherwise, some inline codes will not accept the `ltr` style.

So the answer is yes, please keep it as is.